contribution.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to whatsapp-play
+
+:tada::smiley:First off, thanks for taking the time to contribute! :tada::smiley:
+
+The following is a set of guidelines for contributing to whatsapp-play, on GitHub. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+Whatsapp-play is project that uses python. 
+
+## How Can I Contribute?
+In order to contribute to this project first fork the repo and clone the fork. Play with whatsapp-play at first. If you notice some bugs while playing with it, report the bugs. Check if it is real. You need to follow the following guidelines:
+  1. Raise an issue before sending any PR.
```

Comment:
I think you will face an issue in Indentation of the points you have mentioned
you have mentioned them in : 
1.
2.
3.
rohit's Contribution guidelines says that you have to do it in : 
1.
1.
1.
It will automatically formatted into 1.,2.,3.,.. and so on